subsys/bluetooth/host/gatt.c

```diff
@@ -2469,13 +2469,25 @@ static int gatt_notify(struct bt_conn *conn, uint16_t handle,
 	return bt_att_send(conn, buf);
 }
 
-static void gatt_indicate_rsp(struct bt_conn *conn, uint8_t err,
+/* Converts error (negative errno) to ATT Error code */
+static uint8_t att_err_from_int(int err)
+{
+	LOG_DBG("%d", err);
+
+	if (!IN_RANGE(err, 0, UINT8_MAX)) {
+		return BT_ATT_ERR_UNLIKELY;
+	}
```

Comment:
Please add a comment stating why we are "clamping" the `err` here

Reply:
That's nitpick, come on! If one does not know the size of ATT Error it's better for him to not look into the code.

There's function description in line 2472 saying:
`/* Converts error (negative errno) to ATT Error code */`

Reply:
Nah, I just pushed version with the comment as you suggested. You won ^^